app/assets/javascripts/client/views/assessments/permissions/manage_permission.html

```diff
@@ -60,7 +60,7 @@
             <p class="info">
               <fieldset>
                 <label class="level_label">Current Permission Level: </label>
-                <select class="permission-level" name="level">
+                <select class="permission-level" name="level" ng-disabled="user.role === 'network_partner'">
```

Comment:
In that vein we may be missing something here:  we should sanitize the output list that we get back from the server so that it doesn't have a chance of showing up in the view.

If I remove the `disabled` attribute in my DOM, then I see that the network partner still has other roles.  If we're saying that they should only have the one role, we should sanitize the possible roles that they have.

Observe [permission.rb](https://github.com/MobilityLabs/pdredesign-server/blob/59-permissions/lib/assessments/permission.rb#L12); this is where the JSON to send across the wire is being generated.  We'd want to remove facilitator from this list somehow so we don't have to do anything extra in the view to do so.